sound/soc/amd/acp-config.c

```diff
@@ -34,6 +34,18 @@ static const struct config_entry config_table[] = {
 			{}
 		},
 	},
+	{
+		.flags = FLAG_AMD_SOF,
+		.device = ACP_PCI_DEV_ID,
```

Comment:
just to confirm, this would switch existing solutions for this PCI ID 0x15E2 to SOF. Is this intentional?

amd/mach-config.h:#define ACP_PCI_DEV_ID                        0x15E2
amd/renoir/rn_acp3x.h:#define ACP_DEVICE_ID 0x15E2
amd/vangogh/acp5x.h:#define ACP_DEVICE_ID 0x15E2

Reply:
> We are also planning to add additional checks in pci_probe of these solutions to avoid conflicts with same pci_id.

Why not do it in the common module called by  each pci_probe? That was the recommended solution from Takashi and Jaroslav.